src/BSON/Value.stub.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+/**
+ * @generate-class-entries static
+ * @generate-function-entries static
+ */
+
+namespace MongoDB\BSON;
+
+final class Value
+{
+    /**
+     * @var int
+     * @cvalue BSON_TYPE_EOD
+     */
+    public const TYPE_EOD = 0x00;
+
+    /**
+     * @var int
+     * @cvalue BSON_TYPE_DOUBLE
+     */
+    public const TYPE_DOUBLE = 0x01;
```

Comment:
A note on the values: in other stub files we've used `UNKNOWN` as value whenever we use `@cvalue` to fetch the value from a C definition. Since I was lazy and copied over the declaration from bson-types.h I was able to include the values. There is no downside to this TBH: the generated arginfo has a `ZEND_ASSERT` call to ensure that the value contained in the `@cvalue` member is the same as defined here. The advantage is that when we start using gen_stub.php to generate class synopsis the documentation will contain the correct value for the constant.